sbi/inference/posteriors/base_posterior.py

```diff
@@ -38,6 +42,12 @@ def __init__(
                 `potential_fn.device` is used.
             x_shape: Shape of the observed data.
         """
+        if not isinstance(potential_fn, BasePotential):
+            callable_potential = potential_fn
```

Comment:
do we need this renaming? Can't we just pass `potential_fn` directly to the wrapper? 

Plus, I think it would be good to add a comment here, or explain in the docstring above that `potential_fn` can just a callable (I know, the type has been `Callable` before. Interesting that we never enforced `BasePotential`.)

Reply:
Nope indeed we do not need the renaming.